csm/core/agent/csm_agent.py

```diff
@@ -87,6 +87,12 @@ def init():
         health_service = HealthAppService(health_plugin_obj)
         CsmRestApi._app[const.HEALTH_SERVICE] = health_service
 
+        # Cluster Operations configuration
```

Comment:
This fucntion is getting bulkier. extract this and helth plugin initialization as a separate fucntion

Reply:
Refactored this logic into a new function.